frontend/src/components/review/WIPComment.tsx

```diff
@@ -6,17 +6,28 @@ import {
   CreateCommentResponse,
   CREATE_COMMMENT,
 } from "../../APIClients/mutations/CommentMutations";
+import { CommentResponse } from "../../APIClients/queries/CommentQueries";
+import { StoryLine } from "../translation/Autosave";
+import { convertStatusTitleCase } from "../../utils/StatusUtils";
 
 export type WIPCommentProps = {
   commentStoryTranslationContentId: number;
   lineIndex: number;
   setCommentLine: (line: number) => void;
+  setComments: (comments: CommentResponse[]) => void;
```

Comment:
nit: group setComments with comments and setTranslatedStoryLines with translatedStoryLines